rct229/ruletest_engine/ruletest_jsons/section15/rule_15_5.json

```diff
@@ -0,0 +1,218 @@
+            "comparison_value": "Expected Value",
+            "rule_dependency": "Rule 15-3 = true and xfrm_type = drytype and xfrm_efficiency in User RMR is greater than Table 8.4.4",
+            "mandatory_rule": 0,
+            "schema_version": "0.0.23"
+        },
+        "rmr_transformations": {
+            "user": {
+                "id": "ashrae229",
+                "ruleset_model_instances": [
+                    {
+                        "id": "RMD 1",
+                        "transformers": [
+                            {
+                                "id": "Transformer_1",
+                                "type": "DRY_TYPE",
+                                "phase": "SINGLE_PHASE",
+                                "capacity": 25000,
+                                "efficiency": 0.99
+                            }
+                        ]
+                    }
+                ]
+            },
+            "baseline": {
+                "id": "ashrae229",
+                "ruleset_model_instances": [
+                    {
+                        "id": "RMD 1",
+                        "transformers": [
+                            {
+                                "id": "Transformer_1",
+                                "type": "DRY_TYPE",
+                                "phase": "SINGLE_PHASE",
+                                "capacity": 25000,
+                                "efficiency": 0.99
+                            }
+                        ]
+                    }
+                ]
+            }
+        }
+    },
+    "rule-15-5-c": {
+        "Section": 15,
+        "Rule": 5,
+        "Test": "c",
+        "test_description": "Dry Three-Phase transformer modeled in the User Model whose efficiency is > than Table 8.4.4. Baseline RMR matches Table 8.4.4",
+        "expected_rule_outcome": "pass",
+        "standard": {
+            "rule_id": "15-5",
+            "ruleset_reference": "-",
+            "rule_description": "Transformer efficiency reported in Baseline RMR equals Table 8.4.4",
+            "applicable_rmr": "Baseline RMR",
+            "rule_assertion": "=",
+            "comparison_value": "Expected Value",
+            "rule_dependency": "Rule 15-3 = true and xfrm_type = drytype and xfrm_efficiency in User RMR is greater than Table 8.4.4",
+            "mandatory_rule": 0,
+            "schema_version": "0.0.23"
+        },
+        "rmr_transformations": {
+            "user": {
+                "id": "ashrae229",
+                "ruleset_model_instances": [
+                    {
+                        "id": "RMD 1",
+                        "transformers": [
+                            {
+                                "id": "Transformer_1",
+                                "type": "DRY_TYPE",
+                                "phase": "THREE_PHASE",
+                                "capacity": 30000,
+                                "efficiency": 0.99
```

Comment:
The efficiency needs to be higher than 0.9928 for THREE_PHASE transformer with 30000 capacity. Otherwise 15-5 will fail the applicability check